README.md

```diff
@@ -11,22 +11,24 @@
 
 # DSP-TOOLS - DaSCH Service Platform Tools
 
-DSP-TOOLS is a command line tool that helps you to interact with the DaSCH Service Platform API. 
+DSP-TOOLS is a command line tool that helps you to interact with the DaSCH Service Platform (DSP) API. 
 This document is intended for developers who want to work with the code of DSP-TOOLS. 
 
 | <center>Hint</center>                                                                                                                                                                |
 |:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| This technical document was written as a guide for developers. For the end user documentation, please consult [https://docs.dasch.swiss](https://docs.dasch.swiss/latest/DSP-TOOLS). |
+| This technical document was written as a guide for developers. For the end user documentation, 
+please consult [https://docs.dasch.swiss](https://docs.dasch.swiss/latest/DSP-TOOLS). |
 
 This README contains basic information for getting started. 
 More details can be found in the 
-[developers documentation](https://docs.dasch.swiss/latest/DSP-TOOLS/developers/index/).
+[developers documentation](https://docs.dasch.swiss/latest/DSP-TOOLS/developers/).
 
 
 
-## Quick start
+## Quick Start
 
-To get started quickly, without reading the details, just execute these commands:
+To get started quickly, without reading the details, just execute these commands (a prior installation of 
+[homebrew](https://brew.sh/) is required):
```

Comment:
1. I would put the line break before the opening parentheses (semantic line breaks)
2. I would capitalize Homebrew